### PR TITLE
Align logo vertically

### PR DESCRIPTION
**Pull request by MPParsley**
This solves #1.

### CODE REVIEW

File: examples/login-link.html [hunk 1]
```diff
@@ -359,8 +359,16 @@
...
+                            </td>
+                            <td valign="center;">
```
Comment (MPParsley) [inline examples/login-link.html]:
```suggestion
                            <td valign="center">
```

File: examples/low-emission-zone-request-submitted.html [hunk 2]
```diff
@@ -359,8 +359,16 @@
...
+                            </td>
+                            <td valign="center;">
```
Comment (MPParsley) [inline examples/low-emission-zone-request-submitted.html]:
```suggestion
                            <td valign="center">
```

File: index.html [hunk 3]
```diff
@@ -359,8 +359,16 @@
...
+                            </td>
+                            <td valign="center;">
```
Comment (MPParsley) [inline index.html]:
```suggestion
                            <td valign="center">
```

**Review by hstandaert — APPROVED** [review]

**Comment by mattiasberlamont** [comment]
Thanks @MPParsley and @hstandaert!

**Pull request merged**